discover/apispecs.go

```diff
@@ -0,0 +1,224 @@
+package discover
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-openapi/loads"
+	"github.com/go-openapi/spec"
+	"github.com/go-openapi/swag"
+	wraperrors "github.com/pkg/errors"
+	"github.com/spf13/viper"
+
+	"github.com/kenjones-cisco/dapperdox/config"
+)
+
+const (
+	visibilityExt = "x-visibility"
+
+	groupByPrivate = "Private Cloud Services"
```

Comment:
for the groupBy* types, these should be done via configuration otherwise this would be specific to our internal use.